CMakeLists.txt

```diff
@@ -16,6 +17,14 @@ if (STATIC_BUILD)
     endif()
 endif()
 
+if (EXTERNAL_CHIPDB)
+    if (NOT DEFINED EXTERNAL_CHIPDB_ROOT)
+        message(STATUS "EXTERNAL_CHIPDB_ROOT not defined using -DEXTERNAL_CHIPDB_ROOT=/path/to/nextpnr. Default to /usr/local/share/nextpnr")
```

Comment:
Thinking a bit more, I think the default should be `${CMAKE_INSTALL_PREFIX}/share/nextpnr`